variables.tf

```diff
@@ -379,6 +418,27 @@ variable "os_disk_type" {
   nullable    = false
 }
 
+variable "outbound_ip_address_ids" {
+  type        = list(any)
+  description = "(Optional) The ID of the Public IP Addresses which should be used for outbound communication for the cluster load balancer."
+  default     = null
+  nullable    = true
+}
+
+variable "outbound_ip_prefix_ids" {
+  type        = list(any)
+  description = "(Optional) The ID of the outbound Public IP Address Prefixes which should be used for the cluster load balancer."
+  default     = null
+  nullable    = true
+}
+
+variable "outbound_ports_allocated" {
+  type        = number
+  description = "(Optional) Number of desired SNAT port for each VM in the clusters load balancer. Must be between 0 and 64000 inclusive. Defaults to 0"
+  default     = 0
+  nullable    = false
```

Comment:
According to the [provider schema](https://github.com/hashicorp/terraform-provider-azurerm/blob/307e6478cf507c41301413475d77574618031db5/internal/services/containers/kubernetes_cluster_resource.go#L791-L796), the `outbound_ports_allocated` is optional, which means we could set it to `null` here.